src/email-helpers.js

```diff
@@ -20,7 +20,7 @@ try {
   // Do not set AWS_SDK_LOAD_CONFIG if aws config file is missing.
 }
 
-module.exports = async function sendEmail(filename, url, sender, recipient, transport, smtphost, smtpport, smtpsecure, smtpusername, smtppassword, subject, note, emailbody) {
+module.exports = async function sendEmail(filename, url, sender, recipient, transport, smtphost, smtpport, smtpsecure, smtpusername, smtppassword, subject, note, emailbody, selfsignedcerts) {
```

Comment:
Might start to make sense to turn this into an object-reference, with an interface.   Naming might be hard, though I could make more sense and less brain-power out of  arguments named `{emailContent, smtpConfig}` with associated interfaces.

This would also mean that further config changes (new options) do not cause change on this method (or at least this method signature